.eslintrc.json

```diff
@@ -0,0 +1,18 @@
+{
+  "env": {
+    "es6": true,
+    "node": true
```

Comment:
Why do we need node? This library can only be run in a browser.

Reply:
Ok, I retry with CLI